kernel/kernel-api/src/test/scala/io/delta/kernel/test/MockFileSystemClientUtils.scala

```diff
@@ -107,6 +109,21 @@ trait MockFileSystemClientUtils extends MockEngineUtils {
       jsonHandler = jsonHandler)
   }
 
+  def createMockFSAndJsonEngineForLastCheckpoint(
+      contents: Seq[FileStatus], lastCheckpointVersion: Optional[java.lang.Long]): Engine = {
+    mockEngine(
+      fileSystemClient = new MockListFromFileSystemClient(listFromProvider(contents)),
+      jsonHandler = if (lastCheckpointVersion.isPresent) {
+        new MockReadLastCheckpointFileJsonHandler(
+          s"$logPath/_last_checkpoint",
+          lastCheckpointVersion.get()
+        )
+      } else {
+        null
+      }
+    )
+  }
+
```

Comment:
Any reason to put this here instead of with the `MockReadLastCheckpointFileJsonHandler`?

Reply:
This is next to all the other `createMockFS____` utilities, thought it would be best here